docs/helpers/env_vars.yaml

```
@@ -1203,6 +1203,167 @@ AUDIT_TRACING_TYPE:
   deprecationVersion: ""
   removalVersion: ""
   deprecationInfo: ""
+AUTH_APP_DEBUG_ADDR:
+  name: AUTH_APP_DEBUG_ADDR
+  defaultValue: 127.0.0.1:9245
+  type: string
+  description: Bind address of the debug server, where metrics, health, config and
+    debug endpoints will be exposed.
+  introductionVersion: '%%NEXT%%'
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+AUTH_APP_DEBUG_PPROF:
+  name: AUTH_APP_DEBUG_PPROF
+  defaultValue: "false"
+  type: bool
+  description: Enables pprof, which can be used for profiling.
+  introductionVersion: '%%NEXT%%'
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+AUTH_APP_DEBUG_TOKEN:
+  name: AUTH_APP_DEBUG_TOKEN
+  defaultValue: ""
+  type: string
+  description: Token to secure the metrics endpoint.
+  introductionVersion: '%%NEXT%%'
+  deprecationVersion: ""
+  removalVersion: ""
+  deprecationInfo: ""
+AUTH_APP_DEBUG_ZPAGES:
+  name: AUTH_APP_DEBUG_ZPAGES
+  defaultValue: "false"
+  type: bool
+  description: Enables zpages, which can  be used for collecting and viewing traces
```

```suggestion
  description: Enables zpages, which can be used for collecting and viewing traces
```

How did this end up with an extra space?